tests/core/test_tracker_stores.py

```diff
@@ -34,6 +34,14 @@ def test_get_or_create():
     assert again.get_slot(slot_key) == slot_val
 
 
+def test_get_or_create():
+    get_or_create_tracker_store(InMemoryTrackerStore(domain))
+
+
+def test_dynamo_get_or_create():
```

Comment:
Add 
```
# noinspection PyPep8Naming
@mock_dynamodb2
```
as annotation to the test. And add `from moto import mock_dynamodb2` to the import section.